data/presets/barrier/motorcycle_barrier.json

```
@@ -0,0 +1,13 @@
+{
+    "icon": "maki-roadblock",
+    "fields": [
+        "access"
+    ],
```

should we add `wheelchair` as suggested by the wiki?

```suggestion
    ],
    "moreFields": [
        "wheelchair"
    ],
```